src/ui/run-info-ui-handler.ts

```diff
@@ -194,7 +192,9 @@ export default class GameInfoUiHandler extends UiHandler {
       } else if (this.runInfo.battleType === BattleType.TRAINER) {
         const tObj = this.runInfo.trainer.toTrainer(this.scene);
         const tObjSpriteKey = tObj.config.getSpriteKey(this.runInfo.trainer.variant === TrainerVariant.FEMALE, false);
+        //const tObjSprite = this.scene.add.image(2, 10, "rival_f");
         const tObjSprite = this.scene.add.sprite(0, 5, tObjSpriteKey);
+        console.log(tObjSpriteKey);
```

Comment:
debugging leftovers?
```suggestion
        const tObjSprite = this.scene.add.sprite(0, 5, tObjSpriteKey);
```